phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexVerificationOutputRepository.java

```diff
@@ -85,6 +88,15 @@
     public static final byte[] PHASE_BEFORE_VALUE = Bytes.toBytes("BEFORE");
     public static final byte[] PHASE_AFTER_VALUE = Bytes.toBytes("AFTER");
 
+
+    public enum IndexVerificationErrorType {
+        INVALID_ROW,
```

Comment:
shall we be specific with INVALID_INDEX_ROW and same for others?

Reply:
Since all of them refer to index rows, and Index is in the name of the enum, isn't that implicit?